python-checks/src/test/java/org/sonar/python/checks/RedundantJumpCheckTest.java

```diff
@@ -29,4 +30,16 @@ public void test() {
     PythonCheckVerifier.verify("src/test/resources/checks/redundantJump.py", new RedundantJumpCheck());
   }
 
+  @Test
+  public void quickFixTest() {
+    String input = "def redundant_jump(x):\n" +
+      "  if x == 1:\n" +
+      "    print(True)\n" +
+      "    return";
+    String expected = "def redundant_jump(x):\n" +
+      "  if x == 1:\n" +
+      "    print(True)\n";
+    PythonQuickFixVerifier.verify(new RedundantJumpCheck(), input, expected);
```

Comment:
Similar to what was done in other tests, can you also add an assertion on the quick fix message?

Reply:
done